packages/g/glpk/xmake.lua

```diff
@@ -0,0 +1,54 @@
+package("glpk")
+    set_homepage("https://www.gnu.org/software/glpk/")
+    set_description("The GLPK (GNU Linear Programming Kit) package is intended for solving large-scale linear programming (LP), mixed integer programming (MIP), and other related problems.")
+    set_license("GPL-3.0")
+
+    set_urls("https://ftp.gnu.org/gnu/glpk/glpk-$(version).tar.gz")
+
+    add_versions("5.0", "4a1013eebb50f728fc601bdd833b0b2870333c3b3e5a816eeba921d95bec6f15")
+
+    add_configs("shared", {description = "Build shared binaries", default = false, type = "boolean"})
+
+    if is_plat("linux") then
+        add_extsources("apt::libglpk-dev")
+    end
+
+    if is_plat("macosx|x86_64", "linux") then
+        add_deps("autoconf", "automake", "libtool", "pkg-config")
```

Comment:
if ./configure file exists, we need not add these deps. they are only for ./autogen.sh to generate configure.